jekyll/_api/v1-reference.md

```diff
@@ -4,72 +4,163 @@ title: CircleCI API v1.1 Reference
+
+"https://circleci.com/api/v1.1"
+
+Where:
+
+<dl>
+  <dt>
+  https://circleci.com - the resource URL for the API being called.
+  </dt>
+  <dt>
+  api - the class being called.
+  </dt>
+  <dt>
+  v1.1 - the API version.
+  </dt>
+</dl>
+
+**Note:** Access to billing functions is only available from the CircleCI application.
+
+### Authentication
+
+The CircleCI API utilizes token-based authentication to manage access to the API server and validate that a user has permission to make API requests. Before you can make an API request, you must first add an API token and then verify that you are authenticated by the API server to make requests. The process to add an API token and have the API server authenticate you is described in the sections below.
+
+#### Add an API Token
+
+To add an API token, perform the steps listed below.
+
+1.  Add an API token from your [account dashboard](https://circleci.com/account/api){:rel="nofollow"}.
+2.  To test it,
+    [View it in your browser](https://circleci.com/api/v1.1/me){:rel="nofollow"}
+    or call the API using `curl`:
+
+```
+$ curl https://circleci.com/api/v1.1/me?circle-token=:token
+```
+3.  You should see a response like the following:
+
+```
+{
+  "user_key_fingerprint" : null,
+  "days_left_in_trial" : -238,
+  "plan" : "p16",
+  "trial_end" : "2011-12-28T22:02:15Z",
+  "basic_email_prefs" : "smart",
+  "admin" : true,
+  "login" : "pbiggar"
+}
+```
+**Note** All API calls are made in the same way, by making standard HTTP calls, using JSON, a content-type, and your API token.
+
+#### Get Authenticated
+
+To be authenticated by the API server, add an API token using your [account dashboard](https://circleci.com/account/api). To use the API token, add it to the
+`circle-token` query param: 
+
+```
+curl https://circleci.com/api/v1.1/me?circle-token=:token
+```
+Alternatively, you can use the API token as the username for HTTP Basic Authentication, by passing the `-u` flag to the `curl` command:
+
+```
+curl -u <circle-token>: https://circleci.com/api/...
+```
+
+**Note** the colon `:` tells `curl` that there's no password.
 
 CircleCI 1.0 and 2.0 are supported by API version `1.1` as documented in the following sections:
 
 * TOC
 {:toc}
 
-## Summary
+### Rate Limiting/Throttling ###
+
+Unlike many other APIs, CircleCI does not currently perform any rate limiting or throttling of API requests to the server, CircleCI may, at its discretion, manually limit the number of requests that can be made to the server if it is determined that the user may be acting inappropriately. 
 
-All CircleCI API endpoints begin with `"https://circleci.com/api/v1.1/"`.
+## Summary of API Endpoints
+
+The CircleCI API consists of the following endpoints, which can be called by making a properly-formatted API request.
 
 <dl>
 <dt>
   GET: /me
 </dt>
 <dd>
-  Provides information about the signed in user.
+  Returns detailed information about the user currently signed into the server.
```

Comment:
About the user who's creds are being using to contact the API. Either the API token or browser cookie.